### DISCUSSION TITLE
Why does padding top and bottom differ?

### DISCUSSION THREAD

**Discussion by kirkegaard** [fontsource/fontsource#378] [Q&A]
When using Cairo i see a big difference in how browsers render the font. It seems like both Safari and Firefox adds additional padding on both top and bottom. Ive even tried hardcoding the line-height but without any success.

Am i doing it wrong or is this expected? Can someone explain whats going on?

_The screenshots are using bootstrap. Not sure if that makes any difference?_

Chrome:
<img width="190" alt="Screenshot 2021-10-19 at 16 11 23" src="https://user-images.githubusercontent.com/43752/137946613-0760061b-d72f-485a-a06a-20c81c1a6deb.png">

Safari / Firefox: 
<img width="197" alt="Screenshot 2021-10-19 at 16 11 09" src="https://user-images.githubusercontent.com/43752/137946616-0e8d765c-3bd5-48e3-854a-53075c4e283f.png">

**Comment by ayuhito** [comment]
I'd bring this up in the [google/fonts](https://github.com/google/fonts) repository rather than here.

**Reply by kirkegaard** [reply]
I actually did but nobody replied so i tried my luck here :) But you're right. Its probably more of a font thing than this library. This is just what we use so i thought i would ask

**Reply by ayuhito** [reply]
Ah, yeah. Apologies, I don't really have a clue either.
